morevna-ep4.msrt

```diff
@@ -116,12 +116,12 @@
 24
 00:03:02,960 --> 00:03:04,800
 [rus] А где же движок?
-[eng] But... where is the engine?
+[eng] But... where's the engine?
```

Comment:
Both are right, I made this change to more closely match the speed of the Russian speech.